basis/io/directories/directories.factor

```diff
@@ -93,6 +85,18 @@ M: object copy-file
 : copy-files-into ( files to -- )
     '[ _ copy-file-into ] each ;
 
+! Moving and renaming files
+HOOK: move-file io-backend ( from to -- )
+
+M: object move-file ( from to -- )
+    dupd copy-file delete-file ;
```

Comment:
Is this implementation here so that your new backend works?

Seems like we should remove it for now since the unix backend and the windows backend already define ``move-file``?